app/uk/gov/hmrc/emcstfe/config/AppConfig.scala

```diff
@@ -57,7 +59,9 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig, configuration: Configu
 
   def getFeatureSwitchValue(feature: String): Boolean = configuration.get[Boolean](feature)
 
-  def chrisBaseUrl: String = if(isEnabled(UseChrisStub)) chrisStubUrl else chrisUrl
+  def chrisBaseUrl: String = if(isEnabled(UseChrisStub)) downstreamStubUrl else chrisUrl
```

Comment:
Wouldn't we just call this `isEnabled(DownstreamStub)`? I don't think we'd need two separate flags as it's the one dynamic stub that caters for both.